app/views/sessionNotFound.scala.html

```diff
@@ -0,0 +1,10 @@
+@(message: String)(implicit flash: Flash, request: Request[_])
+
+@main(message) {
+<p class="information">
+    @flash.get("message").fold("")(identity)
+</p>
+
+<h3>Welcome</h3>
+<p>Testing something here.</p>
```

Comment:
Can we use a better heading and text instead of "Welcome" and "Testing something here." ?

Reply:
This file is already changed in #114